### DISCUSSION TITLE
League of Legends client has stopped following the rules set in Hyprland's config and focus_on_activate despite no alterations to the window properties.

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#1924] [Q&A]
For the past week or two the League of Legends client has stopped following the rules declared in Hyprland's config.  In addition, the client has also stopped following the focus_on_activate option. These used to work, and I have not altered their respective Hyprland config entries since. 

With or withour rules it now launches in a size of 112x27. 
I can decrease the size with resizewindow but cannot increase it. 

Hyprctl clients output:
```
Window 3384a70 -> League of Legends:
	mapped: 1
	hidden: 0
	at: 1249,345
	size: 112,27
	workspace: 2 (2)
	floating: 1
	monitor: 0
	class: leagueclientux.exe
	title: League of Legends
	pid: 3570
	xwayland: 1
	pinned: 0
	fullscreen: 0
	fullscreenmode: 0
	fakefullscreen: 0
	grouped: 0
	swallowing: 0
```

I have tried windowrule and windowrulev2 - neither are working. 

```
    windowrule = workspace 2 silent, center, float, size 1280 720, leagueclientux.exe
    windowrulev2 = workspace 2 silent,center,float,size 1280 720, class:^(leagueclientux.exe)$,title:^(League of Legends)$
```

Any ideas what's causing this?

**Comment by ianspy1** [comment]
Hey there, trying to install league with Lutris atm.
Have this weird issue of it being 2 white boxes that I also cant drag bigger.

There seems to be a fix on reddit. BUT. Since that hole reddit strike is going on I am also a bit lost:https://www.reddit.com/r/leagueoflinux/comments/zi4ryj/is_anyone_here_able_to_play_league_on_hyprland/

I also found a video, but I dont speak russian :,) :
https://www.youtube.com/watch?v=sGPR0jUhx7s

**Comment by ianspy1** [comment]
Update ! I found a work arround, although it is janky...

If you go into:
- Lutris
- Arrow up next to wine symbol
- Graphics
- Set Virtual Desktop to your screenres
- Press tab 5 times (this will land you on "apply")

This may need to be done each time you restart lutris, as mine is not saving it propperly.
